Makefile

```diff
@@ -51,7 +51,7 @@ clean:  ## clean up build
 build: build-linux build-mac build-windows  ## build for all targets
 
 .PHONY: build-linux
-build-linux: build-linux-amd64 build-linux-arm64 build-linux-arm-v7 build-linux-s390x  ## build all linux architectures
+build-linux: build-linux-amd64 build-linux-arm64 build-linux-arm-v7 build-linux-s390x build-linux-ppc64 build-linux-ppc64 build-linux-riscv64 ## build all linux architectures
```

Comment:
If we were to do this, it seems to me we should only support amd64 and arm64 here and have another target like build-linux-all with everything else

Reply:
`build-linux-ppc64` is duplicated